.travis.yml

```diff
@@ -7,7 +7,7 @@ env:
  - INTEGRATION=true ES_VERSION=1.7.6 TEST_DEBUG=true
  - INTEGRATION=true ES_VERSION=2.4.4 TEST_DEBUG=true
  - INTEGRATION=true ES_VERSION=5.5.0 TEST_DEBUG=true
- - INTEGRATION=true ES_VERSION=master TEST_DEBUG=true
+ - INTEGRATION=true ES_VERSION=6.x TEST_DEBUG=true
```

Comment:
@andrewvc this will still fail because there is no `6.x` version. It needs to be `6.0.0-beta1` I think